### PR TITLE
livesplit: Add version 1.8.16

### PR DESCRIPTION
**Pull request by Vngdv**

### CODE REVIEW

File: bucket/livesplit.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.8.16",
+    "description": "Speedrun Timer",
+    "homepage": "https://livesplit.org/",
```
Comment (Ash258) [inline bucket/livesplit.json:4]:
```suggestion
    "description": "Timer for speedrunners",
    "homepage": "https://livesplit.org",
```

File: bucket/livesplit.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "license": "MIT",
+    "url": "https://github.com/LiveSplit/LiveSplit/releases/download/1.8.16/LiveSplit_1.8.16.zip",
+    "hash": "83472e16ca536506df23b992013a2af3361474998d4e77cfa4f7cb049172f7ac",
+    "bin": "LiveSplit.exe",
```
Comment (Ash258) [inline bucket/livesplit.json:8]:
```suggestion
    "bin": "LiveSplit.exe",
    "shortcuts": [
        [
            "LiveSplit.exe",
            "LiveSplit"
        ]
    ],
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
Investigate all required files/folders for persisting

https://livesplit.org/downloads/

**Comment by Ash258** [comment]
Why do you request review when you did not do all the required actions??

image

**Comment by Vngdv** [comment]
Oh sorry. I didn't quite understand what you meant. I am going to fix it ASAP!

**Comment by Vngdv** [comment]
So it seems like LiveSplit saves data into "settings.cfg" and "LiveSplit.AutoSplitters.xml" files. They are both contained in the main application folder. I tried to add them like this: `"persist": [ "settings.cfg", "LiveSplit.AutoSplitters.xml" ],`. But this only seemed to work on folders. Did I do something wrong or is there another way to save files?

**Pull request closed without merging**